iam/api-client/quickstartv2.py

```diff
@@ -117,8 +117,9 @@ def set_policy(crm_service, project_id, policy):
 if __name__ == '__main__':
 
     # TODO: replace with your project ID
-    project_id = "your_project_id"
-    # TODO: Replace with the ID of your member in the form 'user:member@example.com.
-    member = "your_member"
+    project_id = "your-project-id"
+    # TODO: Replace with the ID of your member in the form
+    # 'user|group|serviceAccount:email' or 'domain:domain'
```

Comment:
Can we add a link to the IAM reference documentation for this format?

Reply:
The link has some good examples of what `member` should look like.